check-module-version/README.md

```diff
@@ -0,0 +1,42 @@
+# GitHub Action: Check module version
+
+This action checks the equality of the version from Lua 
+module's code and the last repository tag.
```

Comment:
Add a comment that this CI action works only on tag push event.

Reply:
Added this info to both files